compiler/plugins/target/AMD-AIE/aie/CMakeLists.txt

```diff
@@ -287,6 +297,7 @@ iree_cc_library(
     AMDAIEXToStandard.cpp
   DEPS
```

Comment:
Shouldn't `AMDAIEDialect` be added to the dependencies as well?

Reply:
There's no use of `AMDAIEDialect` here yet.

Reply:
Right, nvm